### PR TITLE
Fixing GS guide and links

### PR DESCRIPTION
**Pull request by runleonarun**
## Description & motivation
* Renamed GS file ()
* Fixed links to getting-started-dbt-cloud to include "getting-started folder
* Fixed metadata links

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Comment by jasnonaz** [comment]
This is looking great @runleonarun 
Two questions here:
1. Would it be possible to make the left nav item be "Getting started with dbt" without clicking in? Right now feel like it would be non-obvious where to find this 
image
2. Could we have a top level page that shows Cloud / Core options. We're linking to getting started from the homepage and I would prefer to link to a page that has an option for Cloud or Core rather than linking directly to one.

File: _redirects [hunk 1]
```diff
@@ -467,6 +467,25 @@ docs/contributing/documenting-a-new-adapter /docs/contributing/adapter-developme
 /reference/warehouse-profiles/firebolt-profile /reference/warehouse-setups/firebolt-setup 302
 /reference/warehouse-profiles/mysql-profile /reference/warehouse-setups/mysql-setup 302
 /reference/warehouse-profiles/hive-profile /reference/warehouse-setups/hive-setup 302
+
+# getting started guide
+guides/getting-started /docs/get-started/getting-started/overview 301
```
Comment (runleonarun) [inline _redirects]:
```suggestion
/guides/getting-started /docs/get-started/getting-started/overview 301
```

File: website/docs/docs/build/sql-models.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ id: "sql-models"
 
 :::info Building your first models
 
-If you're new to dbt, we recommend that you check out our [Getting Started guide](/docs/develop/getting-started) to build your first dbt project with models.
+If you're new to dbt, we recommend that you check out our [Getting Started guide](/docs/get-started/overview) to build your first dbt project with models.
```
Comment (runleonarun) [inline website/docs/docs/build/sql-models.md]:
```suggestion
If you're new to dbt, we recommend that you read the [Getting Started guide](/docs/get-started/overview) to build your first dbt project with models.
```

File: _redirects [hunk 3]
```diff
@@ -467,6 +467,25 @@ docs/contributing/documenting-a-new-adapter /docs/contributing/adapter-developme
...
+/guides/getting-started/building-your-first-project/build-your-first-models /docs/get-started/getting-started/building-your-first-project/build-your-first-models 301
+/guides/getting-started/building-your-first-project/schedule-a-job /docs/get-started/getting-started/building-your-first-project/schedule-a-job 301
+/guides/getting-started/building-your-first-project/test-and-document-your-project /docs/get-started/getting-started/building-your-first-project/test-and-document-your-project 301
+/guides/getting-started/create-a-project /docs/get-started/getting-started/create-a-project 301
+/guides/getting-started/getting-set-up /docs/get-started/getting-started/set-up-dbt-cloud 301
+/guides/getting-started/getting-set-up/setting-up-bigquery /docs/get-started/getting-started/getting-set-up/setting-up-bigquery 301
+/guides/getting-started/getting-set-up/setting-up-databricks /docs/get-started/getting-started/getting-set-up/setting-up-databricks 301
+/guides/getting-started/getting-set-up/setting-up-redshift /docs/get-started/getting-started/getting-set-up/setting-up-redshift 301
```
Comment (joellabes) [inline _redirects:481]: I checked one of these redirected properly, I'm trusting you on the rest

File: website/docs/docs/collaborate/documentation.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ id: "documentation"
 * [Declaring properties](configs-and-properties)
 * [`dbt docs` command](cmd-docs)
 * [`doc` Jinja function](/reference/dbt-jinja-functions)
-* If you're new to dbt, we recommend that you check out our [Getting Started Tutorial](/docs/develop/getting-started/getting-set-up) to build your first dbt project, complete with documentation.
+* If you're new to dbt, we recommend that you check out our [Getting Started Tutorial](/docs/get-started/getting-started/set-up-dbt-cloud) to build your first dbt project, complete with documentation.
```
Comment (joellabes) [inline website/docs/docs/collaborate/documentation.md]: Why does this one point to the Cloud page? 
```suggestion
* If you're new to dbt, we recommend that you check out our [Getting Started Tutorial](/docs/get-started/getting-started/overview) to build your first dbt project, complete with documentation.
```

File: website/docs/docs/get-started/getting-started/overview.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+<Card
+    title="Get started with dbt Cloud"
+    body="dbt Cloud is the fastest and most reliable way to deploy dbt. Develop, test, schedule, and investigate data models all in one web-based UI."
+link="/docs/get-started/getting-started/set-up-dbt-cloud"
+    icon="writing"
+/>
+
+<Card
+    title="Getting started with dbt Core"
+    body="dbt Core is an open-source tool that enables data teams to transform data using analytics engineering best practices. "
```
Comment (joellabes) [inline website/docs/docs/get-started/getting-started/overview.md]:
```suggestion
    body="When you use dbt Core to work with dbt, you will be editing files locally using a code editor, and running projects using a command line interface"
```
I think this helps clarify the difference between the two choices

File: website/docs/faqs/Models/create-dependencies.md [hunk 6]
```diff
@@ -44,4 +44,4 @@ Found 2 models, 28 tests, 0 snapshots, 0 analyses, 130 macros, 0 operations, 0 s
 Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
 ```
 
-To learn more about building a dbt project, we recommend you complete the [Getting Started guide](/docs/develop/getting-started).
+To learn more about building a dbt project, we recommend you complete the [Getting Started guide](/docs/get-started/getting-started/set-up-dbt-cloud).
```
Comment (joellabes) [inline website/docs/faqs/Models/create-dependencies.md]: Should this one be pointing to the Cloud variant?
```suggestion
To learn more about building a dbt project, we recommend you complete the [Getting Started guide](/docs/get-started/getting-started/overview).
```

File: website/docs/faqs/Project/example-projects.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ id: example-projects
 
 Yes!
 
-* **Getting Started Tutorial:** You can build your own example dbt project in the [Getting Started Tutorial](/docs/develop/getting-started)
+* **Getting Started Tutorial:** You can build your own example dbt project in the [Getting Started guide](/docs/get-started/getting-started/set-up-dbt-cloud)
```
Comment (joellabes) [inline website/docs/faqs/Project/example-projects.md]: Another one 
```suggestion
* **Getting Started Tutorial:** You can build your own example dbt project in the [Getting Started guide](/docs/get-started/getting-started/overview)
```

File: website/docs/guides/legacy/getting-help.md [hunk 8]
```diff
@@ -7,7 +7,7 @@ dbt is open source, and has a generous community behind it. Asking questions wel
 ### 1. Try to solve your problem first before asking for help
 
 #### Search the existing documentation
-The docs site you're on is highly searchable, make sure to explore for the answer here as a first step. If you're new to dbt, try working through the [Getting Started guide](/docs/develop/getting-started/) first to get a firm foundation on the essential concepts.
+The docs site you're on is highly searchable, make sure to explore for the answer here as a first step. If you're new to dbt, try working through the [Getting Started guide](/docs/get-started/getting-started/) first to get a firm foundation on the essential concepts.
```
Comment (joellabes) [inline website/docs/guides/legacy/getting-help.md]: This path doesn’t exist anymore

```suggestion
The docs site you're on is highly searchable, make sure to explore for the answer here as a first step. If you're new to dbt, try working through the [Getting Started guide](/docs/get-started/getting-started/overview) first to get a firm foundation on the essential concepts.
```

File: website/docs/reference/dbt-jinja-functions/run_query.md [hunk 9]
```diff
@@ -13,7 +13,7 @@ Returns a [Table](https://agate.readthedocs.io/page/api/table.html) object with
 **Note:** The `run_query` macro will not begin a transaction automatically - if you wish to run your query inside of a transaction, please use `begin` and `commit ` statements as appropriate.
 
 :::info Using run_query for the first time?
-Check out the section of the Getting Started guide on [using Jinja](/docs/develop/getting-started/learning-more/using-jinja#dynamically-retrieve-the-list-of-payment-methods) for an example of working with the results of the `run_query` macro!
+Check out the section of the Getting Started guide on [using Jinja](/docs/get-started/getting-started/learning-more/using-jinja#dynamically-retrieve-the-list-of-payment-methods) for an example of working with the results of the `run_query` macro!
```
Comment (joellabes) [inline website/docs/reference/dbt-jinja-functions/run_query.md]: Your link 404s

```suggestion
Check out the section of the Getting Started guide on [using Jinja](/docs/get-started/learning-more/using-jinja#dynamically-retrieve-the-list-of-payment-methods) for an example of working with the results of the `run_query` macro!
```

This change is a much tidier link though! If we weren't coming so close to the wire I'd make an impassioned pitch for dropping the `getting-started` path on all of these

File: website/docs/terms/cte.md [hunk 10]
```diff
@@ -66,7 +66,7 @@ When people talk about how CTEs can simplify your queries, they specifically mea
 
 #### Establish Structure
 
-In leveraging CTEs, you can break complex code into smaller segments, ultimately helping provide structure to your code. At dbt Labs, we often like to use the [import, logical, and final structure](/docs/develop/getting-started/learning-more/refactoring-legacy-sql#implement-cte-groupings) for CTEs which creates a predictable and organized structure to your dbt models.
+In leveraging CTEs, you can break complex code into smaller segments, ultimately helping provide structure to your code. At dbt Labs, we often like to use the [import, logical, and final structure](/docs/get-started/getting-started/learning-more/refactoring-legacy-sql#implement-cte-groupings) for CTEs which creates a predictable and organized structure to your dbt models.
```
Comment (joellabes) [inline website/docs/terms/cte.md]:
```suggestion
In leveraging CTEs, you can break complex code into smaller segments, ultimately helping provide structure to your code. At dbt Labs, we often like to use the [import, logical, and final structure](/docs/get-started/learning-more/refactoring-legacy-sql#implement-cte-groupings) for CTEs which creates a predictable and organized structure to your dbt models.
```

File: website/docs/terms/cte.md [hunk 11]
```diff
@@ -181,7 +181,7 @@ CTEs are essentially temporary views that can be used throughout a query. They a
 
 If you’re interested in reading more about CTE best practices, check out some of our favorite content around model refactoring and style:
 
-- [Refactoring Legacy SQL to dbt](/docs/develop/getting-started/learning-more/refactoring-legacy-sql#implement-cte-groupings)
+- [Refactoring Legacy SQL to dbt](/docs/get-started/getting-started/learning-more/refactoring-legacy-sql#implement-cte-groupings)
```
Comment (joellabes) [inline website/docs/terms/cte.md]:
```suggestion
- [Refactoring Legacy SQL to dbt](/docs/get-started/learning-more/refactoring-legacy-sql#implement-cte-groupings)
```

File: website/docs/terms/dag.md [hunk 12]
```diff
@@ -108,4 +108,4 @@ Ready to restructure (or create your first) DAG? Check out some of the resources
 - [Data modeling techniques for more modularity](https://www.getdbt.com/analytics-engineering/modular-data-modeling-technique/)
 - [How we structure our dbt projects](https://docs.getdbt.com/guides/best-practices/how-we-structure/1-guide-overview)
 - [How to audit your DAG](https://www.youtube.com/watch?v=5W6VrnHVkCA)
-- [Refactoring legacy SQL to dbt](https://docs.getdbt.com/docs/develop/getting-started/learning-more/refactoring-legacy-sql)
+- [Refactoring legacy SQL to dbt](https://docs.getdbt.com/docs/get-started/getting-started/learning-more/refactoring-legacy-sql)
```
Comment (joellabes) [inline website/docs/terms/dag.md]:
```suggestion
- [Refactoring legacy SQL to dbt](/docs/get-started/learning-more/refactoring-legacy-sql)
```

File: website/docs/docs/get-started/getting-started-dbt-core.md [hunk 13]
```diff
@@ -5,13 +5,13 @@ description: "Connecting your warehouse to dbt Core using the CLI."
 sidebar_label: "Getting started"
 ---
 
-When you use dbt Core to work with dbt, you will be editing files locally using a code editor, and running projects using the dbt command line interface (dbt CLI). If you'd rather edit files and run projects using the web-based Integrated Development Environment (IDE), you should refer to [Getting set up in dbt Cloud](/docs/develop/getting-started/getting-set-up).
+When you use dbt Core to work with dbt, you will be editing files locally using a code editor, and running projects using the dbt command line interface (dbt CLI). If you'd rather edit files and run projects using the web-based Integrated Development Environment (IDE), you should refer to [Getting set up in dbt Cloud](/docs/get-started/getting-started/set-up-dbt-cloud).
 
 ## Prerequisites
 
 * To use the dbt CLI, it's important that you know some basics of the Terminal. In particular, you should understand `cd`, `ls` and `pwd` to navigate through the directory structure of your computer easily.
 * Install dbt Core using the [installation instructions](/dbt-cli/install/overview) for your operating system.
```
Comment (joellabes) [inline website/docs/docs/get-started/getting-started-dbt-core.md]: [This](https://deploy-preview-2186--docs-getdbt-com.netlify.app/dbt-cli/install/overview) redirects to https://deploy-preview-2186--docs-getdbt-com.netlify.app/docs/develop/installation which 404s

**Review by joellabes — DISMISSED** [review]
Found a handful of broken links. Also noticed that the installation guides are all giving red links: https://deploy-preview-2186--docs-getdbt-com.netlify.app/docs/get-started/installation

image

File: website/docs/docs/get-started/getting-started-dbt-core.md (hunk 13 above)
Comment (runleonarun) [reply website/docs/docs/get-started/getting-started-dbt-core.md]: Good catch! I fixed the redirects in [this commit](https://github.com/dbt-labs/docs.getdbt.com/pull/2186/commits/df78dcde6b6af084c2c5fad03c844fd4fe635e7e)

**Comment by runleonarun** [comment]
@nghi-ly  and @joellabes This PR is ready for a re-review. Here are the changes I've made since Joel's review:
https://github.com/dbt-labs/docs.getdbt.com/pull/2186/files/fd1f6eec458ca6350eeb4c068c1523b26fb61ba6..b51df6d0cbfa66e2730681eef2594d86c8931050

File: website/docs/docs/dbt-cloud/cloud-ide/ide-beta.md [hunk 14]
```diff
@@ -29,10 +29,10 @@ The IDE is a single interface for building, testing, running, and version contro
 
 To use the dbt Cloud IDE, you need to log in with a dbt Cloud account and click  **Develop** at the top of the page.
 
-You can refer to [Getting Started with dbt Cloud](/docs/develop/getting-started) to quickly get set up and perform some key tasks. For more information, see the following articles.
+You can refer to [Getting Started with dbt Cloud](/docs/get-started/getting-started/set-up-dbt-cloud) to quickly get set up and perform some key tasks. For more information, see the following articles.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-ide/ide-beta.md]:
```suggestion
You can refer to [Getting Started with dbt Cloud](/docs/get-started/getting-started/set-up-dbt-cloud) to quickly get set up and perform some key tasks. For more information, see the following articles:
```

File: website/docs/docs/collaborate/git/connect-azure-devops.md [hunk 15]
```diff
@@ -18,5 +18,5 @@ You can connect your Azure DevOps account in dbt Cloud to unlock new product exp
 
 To connect Azure DevOps in dbt Cloud:
 
-1. An account admin needs to [set up an Active Directory application and add it to dbt Cloud](/docs/dbt-cloud/cloud-configuring-dbt-cloud/setup-azure).
-2. dbt Cloud developers need to [personally authenticate with Azure DevOps](/docs/dbt-cloud/cloud-configuring-dbt-cloud/authenticate-azure) from dbt Cloud.
+1. An account admin needs to [set up an Active Directory application and add it to dbt Cloud](docs/collaborate/git/setup-azure).
```
Comment (nghi-ly) [inline website/docs/docs/collaborate/git/connect-azure-devops.md]:
```suggestion
1. An account admin needs to [set up an Active Directory application and add it to dbt Cloud](/docs/collaborate/git/setup-azure).
```

File: website/docs/docs/collaborate/git/connect-azure-devops.md [hunk 16]
```diff
@@ -18,5 +18,5 @@ You can connect your Azure DevOps account in dbt Cloud to unlock new product exp
...
+2. dbt Cloud developers need to [personally authenticate with Azure DevOps](docs/collaborate/git/authenticate-azure) from dbt Cloud.
```
Comment (nghi-ly) [inline website/docs/docs/collaborate/git/connect-azure-devops.md]:
```suggestion
2. dbt Cloud developers need to [personally authenticate with Azure DevOps](/docs/collaborate/git/authenticate-azure) from dbt Cloud.
```

File: website/docs/guides/migration/versions/05-upgrading-to-v1.3.md [hunk 17]
```diff
@@ -4,14 +4,14 @@ title: "Upgrading to v1.3 (latest)"
...
-- [Cloud upgrade guide](/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-choosing-a-dbt-version)
+- [CLI Installation guide](/docs/get-started/installation)
+- [Cloud upgrade guide](/docs/dbt-versions/upgrade-core-in-cloud)
 
 ## Breaking changes
 
 There are no breaking changes for code in dbt projects and packages. We are committed to providing backward compatibility for all versions 1.x. If you encounter an error upon upgrading, please let us know by [opening an issue](https://github.com/dbt-labs/dbt-core/issues/new).
 
-**Note:** If you have custom code accessing the `raw_sql` property of models (via the [`model`](dbt-jinja-functions/model) or [`graph`](https://docs.getdbt.com/reference/dbt-jinja-functions/graph) objects), it has been renamed to `raw_code`. This is a change to the manifest contract, described in more detail below.
+**Note:** If you have custom code accessing the `raw_sql` property of models (via the [`model`](dbt-jinja-functions/model) or [`graph`](/reference/dbt-jinja-functions/graph) objects), it has been renamed to `raw_code`. This is a change to the manifest contract, described in more detail below.
```
Comment (nghi-ly) [inline website/docs/guides/migration/versions/05-upgrading-to-v1.3.md]:
```suggestion
**Note:** If you have custom code accessing the `raw_sql` property of models (with the [model](dbt-jinja-functions/model) or [graph](/reference/dbt-jinja-functions/graph) objects), it has been renamed to `raw_code`. This is a change to the manifest contract, described in more detail below.
```

File: website/docs/docs/get-started/getting-started/overview.md [hunk 18]
```diff
@@ -26,7 +26,7 @@ link="/docs/get-started/getting-started/set-up-dbt-cloud"
 
 <Card
     title="Getting started with dbt Core"
-    body="dbt Core is an open-source tool that enables data teams to transform data using analytics engineering best practices. "
+    body="When you use dbt Core to work with dbt, you will be editing files locally using a code editor, and running projects using a command line interface"
```
Comment (nghi-ly) [inline website/docs/docs/get-started/getting-started/overview.md:28]: might just be a Netlify thing but the card icons aren't rendering anymore: 

Screen Shot 2022-10-12 at 3 13 21 PM

**Review by nghi-ly — COMMENTED** [review]
looks great, @runleonarun ! a handful of nits. also, seems like the card icon isn't rendering in Deploy Preview

**Review by nghi-ly — APPROVED** [review]
card icons are amazing!

**Pull request merged**